prov/net/src/xnet_eq.c

```diff
@@ -118,6 +124,52 @@ static struct fi_ops xnet_eq_fi_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static int xnet_eq_wait_try_func(void *arg)
+{
+	(void)arg;
+	return FI_SUCCESS;
+}
+
+int xnet_eq_add_progress(struct xnet_eq *eq, struct xnet_progress *progress,
+			 void *context)
+{
+	return ofi_wait_add_fd(eq->util_eq.wait,
+			       ofi_dynpoll_get_fd(&progress->epoll_fd),
+			       POLLIN, xnet_eq_wait_try_func, NULL, context);
+}
+
+int xnet_eq_del_progress(struct xnet_eq *eq, struct xnet_progress *progress)
+{
+	return ofi_wait_del_fd(eq->util_eq.wait,
+			       ofi_dynpoll_get_fd(&progress->epoll_fd));
+}
+
+static int xnet_eq_monitor_all(struct xnet_eq *eq, struct xnet_fabric *fabric)
+{
+	struct xnet_domain *domain;
+	struct dlist_entry *item;
+	int ret;
+
+	ret = xnet_eq_add_progress(eq, &fabric->progress,
+				   &fabric->util_fabric.fabric_fid);
+	if (ret)
+	    return ret;
```

Comment:
Using spaces instead of tab before return

Reply:
My editor was misconfigured. Sorry for all the coding style issues.